eng/vcvars.ps1

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+if ($IsWindows) {
+    # find VS root
+    $vswhere = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe"
+    $visualStudioPath = & $vswhere -prerelease -latest -property installationPath
+    Write-Output "vs located at: $visualStudioPath"
+
+    # Call vcvars64.bat and write the set calls to file
+    cmd.exe /c "call `"$visualStudioPath\VC\Auxiliary\Build\vcvars64.bat`" && set > %temp%\vcvars.txt"
```

Comment:
(I don't know PS) I assume this is equivalent to string escaping in C:
```suggestion
    cmd.exe /c "call \"$visualStudioPath\VC\Auxiliary\Build\vcvars64.bat\" && set > %temp%\vcvars.txt"
```

Reply:
Yes, those are powershell escapes for the quotes

Reply:
@kuzminrobin: If you're curious, they chose `` ` `` as the escape character to avoid clashing with `/` and `\` in paths.